nix/module.nix

```
@@ -203,6 +203,9 @@ in
       bar.customModules.hypridle.rightClick = mkStrOption "";
       bar.customModules.hypridle.scrollDown = mkStrOption "";
       bar.customModules.hypridle.scrollUp = mkStrOption "";
+      bar.customModules.hypridle.startCommand = mkStrOption "nohup hypridle > /dev/null 2>&1 &";
+      bar.customModules.hypridle.stopCommand = mkStrOption "pkill hypridle";
+      bar.customModules.hypridle.isActiveCommand = mkStrOption "pgrep -x 'hypridle' &>/dev/null && echo 'yes' || echo 'no'";
```

It is safe to assume that hypridle is managed by `programs.hypridle.enable` (== systemd) on NixOS
```suggestion
bar.customModules.hypridle.startCommand = "systemctl --user start hypridle.service";
bar.customModules.hypridle.stopCommand = "systemctl --user stop hypridle.service";
bar.customModules.hypridle.isActiveCommand = "systemctl --user status hypridle.service | grep -q 'Active: active (running)' && echo 'yes' || echo 'no'";
```
```

---

That's what I am talking about, you should default to systemd commands on NixOS

---

Yeah that makes sense. Changed!

---

looks like you forgot to commit this